### ISSUE TITLE
Unable to format a 3.5 720KB floppy disk on 3.5 1.44MB disk drive.

### ISSUE THREAD

**Issue by Fenix770** [86Box/86Box#1202]
**Describe the bug**

The DOS FORMAT utility, throws an error when formatting a 3.5 720KB floppy on a 1.44MB disk drive.
This affects all DOS versions from 3.30 (this was the first version with 1.44MB floppy disk support) to MS-DOS 6.22.
I' did not try Win 9x DOS.

Workaround

1. This bug only occurs when 1.44MB floppy is used. It formats correctly a 720kb disk on 720kb drive.
2. On DOS 4.0 and later is possible to force a 720kb format using the FORMAT command with the /F:720 parameter.

**To Reproduce**
1. Crete a VM with a 1.44MB floppy drive
2. Simply boot DOS
3. Create a new 720kb floppy disk
4. Type FORMAT A: (If the disk drive is A)

**Expected behavior**
It should detect the disk format correctly. DOS 3.2 added support for 720kb floppy disks and added disk format autodetection.
(Can format a 5.25 360kb floppy disk on a  5.25 1.2MB drive, altought I did not test, if it happens on the 5.25 disk format).
Then DOS 3.3 added 1.44MB disk support.

Hard Disk Image:
[10MB HD.zip](https://github.com/86Box/86Box/files/5759658/10MB.HD.zip)

NVR and CFG files:
[NVR & CFG files.zip](https://github.com/86Box/86Box/files/5759659/NVR.CFG.files.zip)

**Screenshots**
![Format error](https://user-images.githubusercontent.com/31600563/103449229-22e8ac80-4c84-11eb-9ad5-42118c5d686d.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - 86Box version: 3.0.0.2759
 - Build type: Regular

**Additional context**
I also have another VM with a 386 and DOS 6.0 and has the same issue.

**Comment by OBattler** [comment]
It's not a bug at all. You have to tell FORMAT to format it as 720k or it's going to just format it as 1.44 MB.

**Comment by Fenix770** [comment]
Today I tested Teledisk to convert a TD0 file to the emulated 720kb floppy disks. I also get an error when start to write the 720kb blank floppy image. There is no option to continue onwards. In also configure The floppy drive as 720kb in both 86box configuration and the setup bios.

86box read the TD0 image file as a floppy disk correctly. But I tried this to test if it works, but unfortunately I don't know if it is related

![Teledisk -bug](https://user-images.githubusercontent.com/31600563/103577516-b9e77b80-4eb3-11eb-827a-784f9aef3a1d.png)

I copied Teledisk in the C:\teledisk directory. The image file to transfer is Lemmings.td0.
To start simply

Here is the hard disk image, it is the same as I uploaded before, but has the Teledisk directory
[10MB.zip](https://github.com/86Box/86Box/files/5766750/10MB.zip)


1. type teledisk at dos prompt and enter
2. chose to convert from image file to disk
3. Select the drive letter of the floppy drive
4. It reads the file correcltly but it finds an error on the floppy image file.

B. I tested a self-extracting file that recreates a a 160kb IBM DOS 1.10 floppy disk. This worked flawlessly. However, the created blank disk image is not recognized unless I format it with /1 /4 /8 parameter. I think DOS 3.31 FORMAT command should recognize it automatically 
 
![160kb disk](https://user-images.githubusercontent.com/31600563/103579216-a7bb0c80-4eb6-11eb-993b-e3280eef3912.png)
